src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/HealthCheckSpecification.groovy

```diff
@@ -34,7 +34,7 @@ class HealthCheckSpecification extends HealthCheckBaseSpecification {
             def zkAssertions = new SoftAssertions()
             def activeColor = getActiveNetworkColor(zk)
             def floodlightRegions = flHelper.getFls()*.region
-            def pathsToCheck = ["connecteddevices", "floodlightrouter", "flowhs", "isllatency", "nbworker",
+            def pathsToCheck = ["connecteddevices", "floodlightrouter", /*"flowhs",*/ "isllatency", "nbworker",
```

Comment:
Don't merge until problem with unexpected zookeeper state is resolved.

Reply:
It is already resolved in 'feature/ha_get_paths' branch